### PR TITLE
Fix go-to-def on an abstract explicit impl of an interface method

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/61268

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
@ToddGrun ptal

File: src/EditorFeatures/CSharpTest/CodeActions/ImplementInterface/ImplementImplicitlyTests.cs [hunk 1]
```diff
@@ -266,8 +266,7 @@ public readonly void Goo1() { }
     [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/70232")]
     public async Task TestMissingWhenAlreadyContainingImpl()
     {
-        await TestMissingAsync(
-            """
+        await TestInRegularAndScriptAsync("""
```
Comment (ToddGrun) [inline src/EditorFeatures/CSharpTest/CodeActions/ImplementInterface/ImplementImplicitlyTests.cs:269]: >TestInRegularAndScriptAsync [](http://example.com/codeflow?start=14&length=27)

nit: test name should probably change

**Review by ToddGrun — APPROVED** [review]
:shipit:

File: src/EditorFeatures/CSharpTest/CodeActions/ImplementInterface/ImplementImplicitlyTests.cs (hunk 1 above)
Comment (CyrusNajmabadi) [reply src/EditorFeatures/CSharpTest/CodeActions/ImplementInterface/ImplementImplicitlyTests.cs:269]: Good point. Well fix in follow-up. The original tray was just validating no crash. The actual behavior is fine either way.

**Pull request merged**